src/parse.js

```diff
@@ -20,24 +20,30 @@ parse.section = function(conf, section) {
 	let sectionUrl;
 	let newTab;
 
+	let collection = section.collection;
+
 	if (section['external-url'] !== undefined) {
 		sectionUrl = section['external-url'].href;
 		newTab =
 			section['external-url']['new-tab'] !== undefined ? section['external-url']['new-tab'] : true;
+		// Use title for collection if it's not set
+		if (collection === undefined) {
```

Comment:
Quick one: why is the literal comparison happening for `undefined`, instead of just doing `if(!collection)`?

Reply:
I was just following the pattern from previous line. I'll update it.